File: D.S.A/arrays/easy/Right-Rotate-by-one.js
function RightRotateByOne(arr){
   
    let n= arr.length;
    let last=arr[n-1];

    for(let i=n-2;i>=0;i--){
        arr[i+1]=arr[i];
    }

    arr[0]=last;

    return arr;
}

console.log(RightRotateByOne([1,2,3,4,5]))

File: Java/dsa/arrays/easy/arrays/RightRotateByOne.java
public class RightRotateByOne {
    public static int[] rightRotateByOne(int[] arr) {
        int n = arr.length;
        int last = arr[n - 1];
        for (int i = n - 2; i >= 0; i--) {
            arr[i + 1] = arr[i];
        }
        arr[0] = last;
        return arr;
    }
}
